### ISSUE TITLE
Update to 1.51.0

### ISSUE THREAD

**Issue by webmaster128** [rust-lang/docker-rust#78]
Rust 1.51.0 was released (and it is great, thank you so much for const generics!). Could you create an update for the Docker image?

**Comment by sfackler** [comment]
Upstream PR: https://github.com/docker-library/official-images/pull/9856

They usually take a couple hours to land, FYI.

**Comment by Fuuzetsu** [comment]
The linked PR was merged and I see 1.51.0 tag on docker hub, tagged about 30 minutes ago. But it doesn't seem usable. If we look at the manifest, it shows no images available on any platform:

```console
$ skopeo inspect --raw docker://registry.hub.docker.com/library/rust:1.51.0
{"manifests":[],"mediaType":"application\/vnd.docker.distribution.manifest.list.v2+json","schemaVersion":2}
$ skopeo inspect --raw docker://registry.hub.docker.com/library/rust:1.50.0
{"manifests":[{"digest":"sha256:a37b2a5ba365b9ad5a1b12ff924adb0c0cc27d994e076bbc488a91befb69bf0b","mediaType":"application\/vnd.docker.distribution.manifest.v2+json","platform":{"architecture":"amd64","os":"linux"},"size":1589},{"digest":"sha256:b318da6dfb3bb048204f3dc520022ee1c23e7da51b4b108b91f57d6b25fd82fc","mediaType":"application\/vnd.docker.distribution.manifest.v2+json","platform":{"architecture":"arm","os":"linux","variant":"v7"},"size":1589},{"digest":"sha256:7f438a51ce9ebde4d328364665aee4f16d47b219fa292cbc6bd55d3b2ef7f4ae","mediaType":"application\/vnd.docker.distribution.manifest.v2+json","platform":{"architecture":"arm64","os":"linux","variant":"v8"},"size":1589},{"digest":"sha256:c121c16359ee3b26b6948c474d7e206b6469ea76cd33c734f93d48297c748272","mediaType":"application\/vnd.docker.distribution.manifest.v2+json","platform":{"architecture":"386","os":"linux"},"size":1590}],"mediaType":"application\/vnd.docker.distribution.manifest.list.v2+json","schemaVersion":2}

```

**Comment by sfackler** [comment]
It is probably still building in the Docker Jenkins system.

**Comment by Fuuzetsu** [comment]
> It is probably still building in the Docker Jenkins system.

I see, thank you. Is there some link I can follow the progress on? Cool if not, we'll just wait a bit and try again.

**Comment by sfackler** [comment]
The amd64 build is here: https://doi-janky.infosiftr.net/job/multiarch/job/amd64/job/rust/

It looks like there's quite a queue unfortunately :( Maybe due to the OpenSSL release today?

**Comment by lnicola** [comment]
Some images landed, but e.g. `rust:alpine` wasn't tagged yet even though `1.51-alpine` is there. And I'm not familiar with Jenkins, but it doesn't seem to do much?

![image](https://user-images.githubusercontent.com/308347/112611868-fc6f8300-8e26-11eb-9f1e-80fdbf6539e5.png)

(has it been waiting 10 hours in "allocate node"?)

**Comment by elpiel** [comment]
I believe I think this issue is resolved? The new 1.51 tags seem to be on docker hub.

PS: Thank you for all your work maintaining the images :heart:

**Comment by webmaster128** [comment]
I think so too. Thanks!
